packages/mattermost-components/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@mattermost/components",
```

Comment:
I'm trying to think if we could come up with a less general name for this. We're going to likely end up with another effort to do lower level, structural components again, so I'm trying to think ahead of time if there's something a less ambiguous name we could have for when those are reintroduced. Perhaps they can also go under this same banner though